### DISCUSSION TITLE
Deploy -> Kubernetes -> Analytics ( Deploy job failing in helm installation task )

### DISCUSSION THREAD

**Discussion by srikanth-6520** [Sunbird-Ed/Community#504] [Bugs] [answered]
Hello @gandham-santhosh , we have tried to deploy "Anlytics" job with tag "release-5.1.0" , job failing in helm installation task.

Jenkins Logs:

Started by user [Rajasimman S]
Rebuilds build #[5]
Checking out git https://github.com/project-sunbird/sunbird-devops.git into /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics@script/74766eba9651ee97692d587da848d14cae2c225f81df644eb5a529f729d5e85a to read kubernetes/pipelines/deploy_core/Jenkinsfile
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics@script/74766eba9651ee97692d587da848d14cae2c225f81df644eb5a529f729d5e85a/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-devops.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from https://github.com/project-sunbird/sunbird-devops.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress --depth=1 -- https://github.com/project-sunbird/sunbird-devops.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.1.0^{commit} # timeout=10
Checking out Revision bee2ee6f39baa087eba22646d7919493552b10e1 (origin/release-5.1.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bee2ee6f39baa087eba22646d7919493552b10e1 # timeout=10
Commit message: "Merge pull request #3793 from project-sunbird/gandham-santhosh-patch-1"
First time build. Skipping changelog.
Loading library deploy-conf@heads/shared-lib
Attempting to resolve heads/shared-lib from remote references...
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git ls-remote -- https://github.com/project-sunbird/sunbird-devops.git # timeout=10
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics@libs/b4ac4d1235c67cfedda9d7bf40936b35bc6929460ef19107abe1b183748d4145/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-devops.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from https://github.com/project-sunbird/sunbird-devops.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --no-tags --progress --depth=1 -- https://github.com/project-sunbird/sunbird-devops.git +refs/heads/*:refs/remotes/origin/* refs/heads/shared-lib # timeout=10
Checking out Revision 778da7ce7f97efd003eed98071921a837d209343 (778da7ce7f97efd003eed98071921a837d209343)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 778da7ce7f97efd003eed98071921a837d209343 # timeout=10
Commit message: "fix: move variables to be accessible across both try blocks"
 > git rev-list --no-walk 778da7ce7f97efd003eed98071921a837d209343 # timeout=10
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [Jenkins](https://jenkins.alex-sb.dev.verlab.io/computer/(built-in)/) in /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics
[Pipeline] {
[Pipeline] stage
[Pipeline] { (checkout public repo)
[Pipeline] echo
Found .git folder. Clearing it..
[Pipeline] sh
+ git clean -fxd
Removing ansible/inventory/env/hosts
Removing ansible/inventory/env/keys/
Removing ansible/inventory/env/secrets.yml
Removing kubernetes/ansible/roles/sunbird-deploy/templates/
Removing kubernetes/helm_charts/core/analytics/bundle/
Removing kubernetes/helm_charts/core/analytics/keys/
Removing kubernetes/helm_charts/core/analytics/values.yaml
Removing kubernetes/opa/analytics/bundle.tar.gz
Removing metadata.json
Skipping repository private/
Removing private@tmp/
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-devops.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from https://github.com/project-sunbird/sunbird-devops.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress --depth=1 -- https://github.com/project-sunbird/sunbird-devops.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.1.0^{commit} # timeout=10
Checking out Revision bee2ee6f39baa087eba22646d7919493552b10e1 (origin/release-5.1.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bee2ee6f39baa087eba22646d7919493552b10e1 # timeout=10
Commit message: "Merge pull request #3793 from project-sunbird/gandham-santhosh-patch-1"
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (deploy)
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] sh
+ echo Deploy/alex/dev/Kubernetes/Analytics
[Pipeline] sh
+ echo Deploy/alex/dev/Kubernetes/Analytics
[Pipeline] sh
+ echo Deploy/alex/dev/Kubernetes/Analytics
[Pipeline] stage
[Pipeline] { (parameter checks)
[Pipeline] echo
Found environment variable named hub_org with value as: 961571523562.dkr.ecr.ap-south-1.amazonaws.com/prod
[Pipeline] copyArtifacts
Copied 1 artifact from "[ArtifactUpload » dev » Core » Analytics](https://jenkins.alex-sb.dev.verlab.io/job/ArtifactUpload/job/dev/job/Core/job/Analytics/)" build number [3](https://jenkins.alex-sb.dev.verlab.io/job/ArtifactUpload/job/dev/job/Core/job/Analytics/3/)
[Pipeline] sh
+ jq -r .image_name metadata.json
[Pipeline] sh
+ jq -r .node_name metadata.json
[Pipeline] echo
image_tag not specified, using the image_tag specified in metadata.json. 
[Pipeline] sh
+ jq -r .image_tag metadata.json
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] sh
+ echo Deploy/alex/dev/Kubernetes/Analytics
[Pipeline] sh
+ pwd
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (ansible-run)
[Pipeline] echo
{env=dev, module=Kubernetes, jobName=Analytics, absolute_job_path=ArtifactUpload/dev/Core/Analytics, agent=built-in, image_name=sunbird-analytics-service, image_tag=release-5.1.0_RC2_a7544a4_2, currentWs=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics, ansiblePlaybook=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/kubernetes/ansible/deploy_core_service.yml, ansibleExtraArgs=--extra-vars "chart_path=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/kubernetes/helm_charts/core/analytics release_name=analytics image_tag=release-5.1.0_RC2_a7544a4_2 image_name=sunbird-analytics-service role_name=sunbird-deploy" --vault-password-file /var/lib/jenkins/secrets/vault-pass}
[Pipeline] dir
Running in /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/private
[Pipeline] {
[Pipeline] echo
Info: Branch override is enabled
[Pipeline] checkout
The recommended git tool is: NONE
using credential bitbucket-cred
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/private/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url bitbucket.org:verlab/veranda-alex-sb-private.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from bitbucket.org:verlab/veranda-alex-sb-private.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
using GIT_SSH to set credentials Veranda Bitbucket Credential (https://bitbucket.org/verlab/)
Verifying host key using known hosts file
 > git fetch --no-tags --progress --depth=1 -- bitbucket.org:verlab/veranda-alex-sb-private.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.1.0-alex-dev^{commit} # timeout=10
Checking out Revision 8d98b7a956db10a7d4f9d044281a55022a61ae98 (origin/release-5.1.0-alex-dev)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 8d98b7a956db10a7d4f9d044281a55022a61ae98 # timeout=10
Commit message: ":up: the following variables updated for Deploy/Kubernetes/AdminUtils - core_vault_sunbird_sso_publickey - adminutil_refresh_token_public_key_kid - adminutil_refresh_token_secret_key"
 > git rev-list --no-walk 863a6120fac5ee2b99534fb8067e70a9e5666983 # timeout=10
[Pipeline] }
[Pipeline] // dir
[Pipeline] sh
+ rsync -Lkr /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/private/ansible/inventory/dev/Kubernetes/common.yml /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/private/ansible/inventory/dev/Kubernetes/hosts /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/private/ansible/inventory/dev/Kubernetes/keys /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/private/ansible/inventory/dev/Kubernetes/secrets.yml /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/
+ [ -f /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/kubernetes.yaml ]
+ ansible-playbook -i /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/hosts /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/kubernetes/ansible/deploy_core_service.yml --extra-vars chart_path=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/kubernetes/helm_charts/core/analytics release_name=analytics image_tag=release-5.1.0_RC2_a7544a4_2 image_name=sunbird-analytics-service role_name=sunbird-deploy --vault-password-file /var/lib/jenkins/secrets/vault-pass
[WARNING]: Invalid characters were found in group names but not replaced, use
-vvvv to see details

[WARNING]:  * Failed to parse /var/lib/jenkins/workspace/Deploy/alex/dev/Kubern
etes/Analytics/ansible/inventory/env/hosts with yaml plugin: Syntax Error while
loading YAML.   found unexpected ':'  The error appears to be in '/var/lib/jenk
ins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/hosts'
: line 1, column 5, but may be elsewhere in the file depending on the exact
syntax problem.  The offending line appears to be:   [all:vars]     ^ here

[WARNING]:  * Failed to parse /var/lib/jenkins/workspace/Deploy/alex/dev/Kubern
etes/Analytics/ansible/inventory/env/hosts with ini plugin: /var/lib/jenkins/wo
rkspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/hosts:150:
Section [core:children] includes undefined group: postgresql-slave

[WARNING]: Unable to parse /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernete
s/Analytics/ansible/inventory/env/hosts as an inventory source

[WARNING]: No inventory was parsed, only implicit localhost is available

[WARNING]: provided hosts list is empty, only localhost is available. Note that
the implicit localhost does not match 'all'

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 3, column 1, found a duplicate dict key
(sunbird_time_zone). Using last defined value only.

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 3, column 1, found a duplicate dict key
(nginx_per_ip_connection_limit). Using last defined value only.

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 3, column 1, found a duplicate dict key
(sunbird_portal_player_cdn_enabled). Using last defined value only.

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 3, column 1, found a duplicate dict key
(dial_liveness_readiness). Using last defined value only.

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 924, column 5, found a duplicate dict key
(failureThreshold). Using last defined value only.

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 934, column 5, found a duplicate dict key
(failureThreshold). Using last defined value only.

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 947, column 5, found a duplicate dict key
(failureThreshold). Using last defined value only.

[WARNING]: While constructing a mapping from /var/lib/jenkins/workspace/Deploy/
alex/dev/Kubernetes/Analytics/kubernetes/ansible/roles/sunbird-
deploy/defaults/main.yml, line 957, column 5, found a duplicate dict key
(failureThreshold). Using last defined value only.

PLAY [local] *******************************************************************

TASK [check for kong db migration] *********************************************
skipping: [localhost]

TASK [sunbird-deploy : template values.yaml file] ******************************
[DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host localhost should use 
/usr/bin/python3, but is using /usr/bin/python for backward compatibility with 
prior Ansible releases. A future Ansible release will default to using the 
discovered platform python for this host. See [https://docs.ansible.com/ansible/](https://docs.ansible.com/ansible/%1B[0m)
2.8/reference_appendices/interpreter_discovery.html for more information. This 
feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
changed: [localhost]

TASK [sunbird-deploy : copy and template env file from swarm role] *************
changed: [localhost] => (item=../../../../ansible/roles/stack-sunbird/templates/analytics_api_service.conf)

TASK [sunbird-deploy : create env configmap] ***********************************
changed: [localhost]

TASK [sunbird-deploy : create aplication configmap] ****************************
changed: [localhost]

TASK [sunbird-deploy : create xml configmap] ***********************************
changed: [localhost]

TASK [Check opa testcases coverage is 100%] ************************************

TASK [opa-test-coverage : Register list of .bak files as a variable] ***********
ok: [localhost]

TASK [opa-test-coverage : Avoid overwrite of files on multiple runs by replacing the templated files with .bak files] ***
skipping: [localhost] => (item=main.rego) 
skipping: [localhost] => (item=common.rego) 

TASK [opa-test-coverage : Keep a copy of the files before templating] **********
changed: [localhost] => (item=main.rego)
changed: [localhost] => (item=common.rego)

TASK [opa-test-coverage : Template the common rego files] **********************
changed: [localhost] => (item=main.rego)
changed: [localhost] => (item=common.rego)

TASK [opa-test-coverage : Run the opa test and check for 100% coverage on service policies] ***
changed: [localhost]

TASK [opa-test-coverage : Run the opa test and check for 100% coverage on common policies] ***
changed: [localhost]

TASK [opa-test-coverage : Check if coverage is 100% from policies and tests files] ***
skipping: [localhost] => (item=policies.rego) 
skipping: [localhost] => (item=policies_test.rego) 

TASK [opa-test-coverage : Check if coverage is 100% from common policies and tests files] ***
skipping: [localhost] => (item=main.rego) 
skipping: [localhost] => (item=common.rego) 

TASK [opa-test-coverage : Replace the templated files with original files] *****
changed: [localhost] => (item=main.rego)
changed: [localhost] => (item=common.rego)

TASK [Load role to decrypt private keys, copy public keys to helm chart] *******

TASK [mount-keys : Decrypt the data first] *************************************
changed: [localhost] => (item=1)
changed: [localhost] => (item=2)
changed: [localhost] => (item=3)
changed: [localhost] => (item=4)
changed: [localhost] => (item=5)
changed: [localhost] => (item=6)
changed: [localhost] => (item=7)
changed: [localhost] => (item=8)
changed: [localhost] => (item=9)
changed: [localhost] => (item=10)

TASK [mount-keys : Generate public keys for private keys] **********************
changed: [localhost] => (item=1)
changed: [localhost] => (item=2)
changed: [localhost] => (item=3)
changed: [localhost] => (item=4)
changed: [localhost] => (item=5)
changed: [localhost] => (item=6)
changed: [localhost] => (item=7)
changed: [localhost] => (item=8)
changed: [localhost] => (item=9)
changed: [localhost] => (item=10)

TASK [mount-keys : Register list of files as a variable] ***********************
ok: [localhost]

TASK [mount-keys : Create a temporary file] ************************************
changed: [localhost]

TASK [mount-keys : Insert the public keys as key:value pairs into the file] ****
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key10)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key1)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key8)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key5)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key7)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key9)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key3)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key4)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key6)
changed: [localhost] => (item=/var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/ansible/inventory/env/keys/accessv1_key2)

TASK [mount-keys : Set a variable with the file contents] **********************
ok: [localhost]

TASK [mount-keys : Register a variable for keycloak public key in PEM format] ***
changed: [localhost]

TASK [mount-keys : Remove the temporary file] **********************************
changed: [localhost]

TASK [mount-keys : Create the keys directory in case user is overriding the private_key_path] ***
changed: [localhost]

TASK [mount-keys : Copy keys directory to chart path] **************************
changed: [localhost] => (item=1)
changed: [localhost] => (item=2)
changed: [localhost] => (item=3)
changed: [localhost] => (item=4)
changed: [localhost] => (item=5)
changed: [localhost] => (item=6)
changed: [localhost] => (item=7)
changed: [localhost] => (item=8)
changed: [localhost] => (item=9)
changed: [localhost] => (item=10)

TASK [sunbird-deploy : Create the token public key file] ***********************
skipping: [localhost]

TASK [sunbird-deploy : Replace ansible variables with values] ******************
changed: [localhost] => (item=main.rego)
changed: [localhost] => (item=common.rego)

TASK [sunbird-deploy : Create OPA bundle] **************************************
[WARNING]: Consider using the file module with state=absent rather than running
'rm'.  If you need to use command because file is insufficient you can add
'warn: false' to this command task or set 'command_warnings=False' in
ansible.cfg to get rid of this message.

changed: [localhost]

TASK [sunbird-deploy : Copy the OPA bundle to helm chart directory] ************
changed: [localhost]

TASK [sunbird-deploy : helm install] *******************************************
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": "helm upgrade --install --atomic --timeout 10m analytics /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Analytics/kubernetes/helm_charts/core/analytics -n alexdev",
    "delta": "0:10:00.606653",
    "end": "2023-05-24 15:15:42.101231",
    "rc": 1,
    "start": "2023-05-24 15:05:41.494578"
}

STDOUT:

Release "analytics" does not exist. Installing it now.

STDERR:

Error: release analytics failed, and has been uninstalled due to atomic being set: timed out waiting for the condition

MSG:

non-zero return code

PLAY RECAP *********************************************************************
localhost                  : ok=24   changed=21   unreachable=0    failed=1    skipped=5    rescued=0    ignored=0   

[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // stage
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (slack_notify)
[Pipeline] sh
+ git rev-parse --short HEAD
[Pipeline] sh
+ git name-rev --name-only HEAD
+ cut -d / -f1
+ rev
+ rev
[Pipeline] sh
+ echo Deploy/alex/dev/Kubernetes/Analytics
[Pipeline] echo
Could not find env specific Slack channel. Check for global slack channel..
[Pipeline] echo
Could not find slack environment variable. Skipping slack notification..
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (email_notify)
[Pipeline] sh
+ echo Deploy/alex/dev/Kubernetes/Analytics
[Pipeline] echo
Could not find env specific email group. Check for global email group..
[Pipeline] echo
Could not find global email group variable. Skipping email notification..
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 2
Finished: FAILURE

**Comment by santhosh-tg** [comment]
@srikanth-6520  Please check the pod logs while the deployment is happening.  The deployment rolls back in 10 mins , so make sure to check before that

**Reply by srikanth-6520** [reply]
@santhosh-tg we have check the pod logs while deploying the job , and there we found error in logs , we have shared the logs with you , please find it below.

analytics WARNING: An illegal reflective access operation has occurred
analytics WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/home/analytics/analytics-api-2.0/lib/com.google.inject.guice-4.2.2.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
analytics WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
analytics WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
analytics WARNING: All illegal access operations will be denied in a future release
analytics Starting DeviceRegisterService
analytics Starting DeviceRegisterService
analytics Starting DeviceRegisterService
analytics Starting DeviceRegisterService
analytics Starting DeviceRegisterService
analytics Starting DeviceRegisterService
analytics starting DeviceProfileService
analytics starting DeviceProfileService
analytics starting DeviceProfileService
analytics starting DeviceProfileService
analytics starting DeviceProfileService
analytics starting DeviceProfileService
analytics [info] play.api.Play - Application started (Prod) (no global state)
analytics [info] p.c.s.NettyServer - Listening for HTTP on /0:0:0:0:0:0:0:0%0:9000
analytics Uncaught error from thread [application-akka.actor.default-dispatcher-9]: 'scala.collection.mutable.ArrayOps scala.Predef$.refArrayOps(java.lang.Object[])', shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[application]
analytics java.lang.NoSuchMethodError: 'scala.collection.mutable.ArrayOps scala.Predef$.refArrayOps(java.lang.Object[])'
analytics     at org.ekstep.analytics.framework.util.CommonUtil$.getMessageId(CommonUtil.scala:491)
analytics     at org.ekstep.analytics.api.util.APILogger$.getAccessMeasuredEvent(APILogger.scala:32)
analytics     at org.ekstep.analytics.api.util.APILogger$.log(APILogger.scala:20)
analytics     at org.ekstep.analytics.api.util.CacheUtil.initDeviceLocationCache(CacheUtil.scala:79)
analytics     at org.ekstep.analytics.api.util.CacheUtil.init(CacheUtil.scala:38)
analytics     at org.ekstep.analytics.api.service.CacheRefreshActor$$anonfun$receive$1.applyOrElse(CacheRefreshActor.scala:23)
analytics     at akka.actor.Actor.aroundReceive(Actor.scala:539)
analytics     at akka.actor.Actor.aroundReceive$(Actor.scala:537)
analytics     at org.ekstep.analytics.api.service.CacheRefreshActor.aroundReceive(CacheRefreshActor.scala:12)
analytics     at akka.actor.ActorCell.receiveMessage(ActorCell.scala:610)
analytics     at akka.actor.ActorCell.invoke(ActorCell.scala:579)
analytics     at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:268)
analytics     at akka.dispatch.Mailbox.run(Mailbox.scala:229)
analytics     at akka.dispatch.Mailbox.exec(Mailbox.scala:241)
analytics     at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
analytics     at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
analytics     at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
analytics     at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
analytics [error] a.a.ActorSystemImpl - Uncaught error from thread [application-akka.actor.default-dispatcher-9]: 'scala.collection.mutable.ArrayOps scala.Predef$.refArrayOps(java.lang.Object[])', shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[application]
analytics java.lang.NoSuchMethodError: 'scala.collection.mutable.ArrayOps scala.Predef$.refArrayOps(java.lang.Object[])'
analytics     at org.ekstep.analytics.framework.util.CommonUtil$.getMessageId(CommonUtil.scala:491)
analytics     at org.ekstep.analytics.api.util.APILogger$.getAccessMeasuredEvent(APILogger.scala:32)
analytics     at org.ekstep.analytics.api.util.APILogger$.log(APILogger.scala:20)
analytics     at org.ekstep.analytics.api.util.CacheUtil.initDeviceLocationCache(CacheUtil.scala:79)
analytics     at org.ekstep.analytics.api.util.CacheUtil.init(CacheUtil.scala:38)
analytics     at org.ekstep.analytics.api.service.CacheRefreshActor$$anonfun$receive$1.applyOrElse(CacheRefreshActor.scala:23)
analytics     at akka.actor.Actor.aroundReceive(Actor.scala:539)
analytics     at akka.actor.Actor.aroundReceive$(Actor.scala:537)
analytics     at org.ekstep.analytics.api.service.CacheRefreshActor.aroundReceive(CacheRefreshActor.scala:12)
analytics     at akka.actor.ActorCell.receiveMessage(ActorCell.scala:610)
analytics DeviceProfileService stopped successfully
analytics DeviceProfileService stopped successfully
analytics DeviceProfileService stopped successfully
analytics DeviceProfileService stopped successfully
analytics DeviceProfileService stopped successfully
analytics DeviceRegisterService stopped successfully
analytics DeviceProfileService stopped successfully
analytics DeviceRegisterService stopped successfully
analytics DeviceRegisterService stopped successfully
analytics DeviceRegisterService stopped successfully
analytics DeviceRegisterService stopped successfully
analytics DeviceRegisterService stopped successfully
Stream closed EOF for alexdev/analytics-5cbc9df9b7-4zr25 (proxy-init)

**Reply by santhosh-tg** [reply]
@manjudr can you help with this issue, please ?

**Reply by srikanth-6520** [reply]
@santhosh-tg we are waiting for you are replay , please kindly help us to resolve the issue.

**Reply by santhosh-tg** [reply]
@anandp504 @manjudr  Request your help to check on this issue.

**Reply by srikanth-6520** [reply]
@santhosh-tg we are waiting for you are replay , please kindly help us to resolve the issue.

**Reply by santhosh-tg** [reply]
@srikanth-6520  As you can see i have  already tagged the respective people for help. You may want to wait for their reply 
@chitranshu-keshav

**Reply by santhosh-tg** [reply]
@anandp504 @manjudr Request you to provide some insights on this issue.

**Reply by srikanth-6520** [reply, marked as answer]
@santhosh-tg issue fixed , thank you.
